### PR TITLE
Latin Abbreviations "vs" Updated to "versus"

### LINKED ISSUE #19179: “Managing Service Account” uses Latin abbreviation in heading
_This is a_ good first issue. _If you have already contributed a pull request to Kubernetes documentation, please consider leaving this fix for someone who is getting started._

**Problem:**
https://kubernetes.io/docs/reference/access-authn-authz/service-accounts-admin/#user-accounts-vs-service-accounts uses Latin text in a heading.
In the documentation style guide, SIG Docs recommends [avoiding Latin phrases](https://kubernetes.io/docs/contribute/style/style-guide/#avoid-latin-phrases). That recommendation is particularly relevant for headings because heading text is more visible.

**Proposed Solution:**
Edit https://github.com/kubernetes/website/blob/master/content/en/docs/reference/access-authn-authz/service-accounts-admin.md to use a new heading. For example, use the word “compare”, or the full word “versus”.

**Page to Update:**
https://kubernetes.io/docs/reference/access-authn-authz/service-accounts-admin/

**Additional Information:**
/good-first-issue
/language en
/priority backlog
/kind cleanup

### PR DESCRIPTION
**Pull request by aallbrig**
fixes #19179

[SIG Docs recommends avoiding Latin phrases](https://kubernetes.io/docs/contribute/style/style-guide/#avoid-latin-phrases). "vs" is a latin abbreviation and I figured I could use `grep`, `sed` and `xargs` to update all markdown files in `content/en/docs` from my OSX work station.

Here is the command used:
``` bash
grep -lR ' vs ' ./content/en/docs | xargs sed -i '' -e 's/ vs / versus /g'
```

### CODE REVIEW

File: content/en/docs/concepts/configuration/overview.md [hunk 1]
```diff
@@ -30,7 +30,7 @@ This is a living document. If you think of something that is not on this list bu
 - Put object descriptions in annotations, to allow better introspection.
 
 
-## "Naked" Pods vs ReplicaSets, Deployments, and Jobs
+## "Naked" Pods versus ReplicaSets, Deployments, and Jobs
```
Comment (sftim) [inline content/en/docs/concepts/configuration/overview.md]: To preserve existing links:
```suggestion
## "Naked" Pods versus ReplicaSets, Deployments, and Jobs {#naked-pods-vs-replicasets-deployments-and-jobs}
```

File: content/en/docs/concepts/policy/resource-quotas.md [hunk 2]
```diff
@@ -376,7 +376,7 @@ pods        0     10
 * `Exist`
 * `DoesNotExist`
 
-## Requests vs Limits
+## Requests versus Limits
```
Comment (sftim) [inline content/en/docs/concepts/policy/resource-quotas.md]: How about:
```suggestion
## Requests compared to Limits {#requests-vs-limits}
```
?

File: content/en/docs/reference/access-authn-authz/service-accounts-admin.md [hunk 3]
```diff
@@ -18,7 +18,7 @@ incomplete features are referred to in order to better describe service accounts
 {{% /capture %}}
 
 {{% capture body %}}
-## User accounts vs service accounts
+## User accounts versus service accounts
```
Comment (sftim) [inline content/en/docs/reference/access-authn-authz/service-accounts-admin.md]: Great!

Comment (aallbrig) [reply content/en/docs/reference/access-authn-authz/service-accounts-admin.md]: :)

**Comment by aallbrig** [comment]
@sftim Thanks for the update requests. I've reviewed each and gotten them committed to this branch.

I feel good about these changes now 👍

**Comment by aallbrig** [comment]
:D woohoo!

**Pull request merged**